tools/deploy.py

```diff
@@ -21,39 +21,69 @@
 {}
 """
 
+DEPENDENCIESPATH = "docs/_includes/dependencies_list.md"
+
 REPOUSER = "acemod"
 REPONAME = "ACE3"
 REPOPATH = "{}/{}".format(REPOUSER,REPONAME)
 
 
-def update_translations(token):
+def update_translations(repo):
     diag = sp.check_output(["python3", "tools/stringtablediag.py", "--markdown"])
     diag = str(diag, "utf-8")
-    repo = Github(token).get_repo(REPOPATH)
     issue = repo.get_issue(TRANSLATIONISSUE)
     issue.edit(body=TRANSLATIONBODY.format(diag))
 
+def update_dependencies(repo):
+    dependencies = sp.check_output(["python3", "tools/extract_dependencies.py", "--markdown"])
+    dependencies = str(dependencies, "utf-8")
+    diff = sp.check_output(["git", "diff", "--name-only", DEPENDENCIESPATH])
+    diff = str(diff, "utf-8")
+
+    if diff != "":
+        sha = repo.get_contents(DEPENDENCIESPATH
+            #, ref="travisForDocs" # Debug
+        ).sha
+        repo.update_file(
+            path="/{}".format(DEPENDENCIESPATH),
+            message="[Docs] Update component dependencies\nAutomatically committed through Travis CI.\n\n[ci skip]",
+            content=dependencies, sha=sha, committer=InputGitAuthor("ace3mod", "ace3mod@gmail.com")
+            #, branch="travisForDocs" # Debug
```

Comment:
I thought I'd leave it there for future, as that's pretty simple to forget, and that way you can just quickly uncomment those and it will run on a branch (with some extra `.travis.yml` editing).